.travis.yml

```diff
@@ -1,47 +1,38 @@
 language: python
-
 branches:
   only:
-    - master
-
+  - master
 services:
-  - docker
+- docker
 sudo: required
-
 before_install:
-  - docker-compose -f .travis/docker-compose-travis.yml up -d
-
+- docker-compose -f .travis/docker-compose-travis.yml up -d
 install:
-    - pip install -r requirements/travis.txt
-
+- pip install -r requirements/travis.txt
 script:
-  - docker exec -t -e TRAVIS=1 analytics_api_testing bash -c "
-      cd /edx/app/analytics_api/analytics_api/ &&
-      make $TARGETS
-      "
-
+- docker exec -t -e TRAVIS=1 analytics_api_testing bash -c " cd /edx/app/analytics_api/analytics_api/
+  && make $TARGETS "
 matrix:
   include:
-    - python: 3.5
-      env:
-        TESTNAME=quality-python-3.5
-        TARGETS="PYTHON_ENV=py35 quality"
-    - python: 3.5
-      env:
-        TESTNAME=test-python-3.5
-        TARGETS="PYTHON_ENV=py35 test"
-      after_success:
-        - docker exec analytics_api_testing /edx/app/analytics_api/analytics_api/.travis/run_coverage.sh
-        - codecov --disable pycov
-
-    - python: 3.8
-      env:
-        TESTNAME=quality-python-3.8
-        TARGETS="PYTHON_ENV=py38 quality"
-    - python: 3.8
-      env:
-        TESTNAME=test-python-3.8
-        TARGETS="PYTHON_ENV=py38 test"
-
+  - python: 3.5
+    env: TESTNAME=quality-python-3.5 TARGETS="PYTHON_ENV=py35 quality"
+  - python: 3.5
+    env: TESTNAME=test-python-3.5 TARGETS="PYTHON_ENV=py35 test"
+    after_success:
+    - docker exec analytics_api_testing /edx/app/analytics_api/analytics_api/.travis/run_coverage.sh
+    - codecov --disable pycov
+  - python: 3.8
```

Comment:
I'm curious why we're running 3.8 or if we should be running other versions, but I think it should stay as is for now.